pom.xml

```diff
@@ -170,6 +169,7 @@
     <pentaho-metaverse.version>10.1.0.0-SNAPSHOT</pentaho-metaverse.version>
     <encryption-support.version>10.1.0.0-SNAPSHOT</encryption-support.version>
     <jakarta-annotation-api.version>1.3.5</jakarta-annotation-api.version>
+    <ws-commons-util-version>1.0.2</ws-commons-util-version>
```

Comment:
Why have you added this property?

Reply:
Compilation error in the day of local build. Already solved. No longer needed